lib/membrane/live_compositor/lifecycle.ex

```diff
@@ -62,11 +62,15 @@ defmodule Membrane.LiveCompositor.Lifecycle do
   """
   @type input_eos :: :input_eos
 
-  @type notification :: {
-          output_registered()
-          | input_registered()
+  @type event ::
```

Comment:
If I remember correctly, you had strong objections to calling those events. I don't really see why you are adding new type, but i don't object to it either